department-of-veterans-affairs/va.gov-cms
# VACMS-13478: contrib module updates for d10 readiness

## Description

Closes #13478.
Closes #13605.
Closes #12187.
Closes #12188.


## Testing done
Tested locally. 

## Screenshots


## QA steps

What needs to be checked to prove this works?  
What needs to be checked to prove it didn't break any related things?  
What variations of circumstances (users, actions, values) need to be checked?  

There should not be any breaking changes in this one, which doesn't mean there aren't. I will use devel generate to generate 50 pieces of content for each content type and then check for errors. Please be picky on this one. 

1. Create a Staff Profile without a Bio
- [x] Validate that it saves
2. Create a staff profile WITH a Bio
- [x] Validate that it saves.

### Definition of Done


### Select Team for PR review



### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
